scripts/build/ci-msi-build/ci-msi-build-oss.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Should probably `set -e` or add the exit_if_fail command that is unsed in some of the other build scripts so that a failing build makes the bash script return a nono-zero exit code and fails the ci build. This goes for all shell scripts.

Reply:
`set -e` has been added